lib/engine/corporation.rb

```diff
@@ -43,10 +44,10 @@ def initialize(sym:, name:, tokens:, **opts)
       @float_percent = opts[:float_percent] || 60
       @coordinates = opts[:coordinates]
       @min_price = opts[:min_price]
-      @logo = "/logos/#{opts[:logo]}.svg"
       @color = opts[:color]
       @text_color = opts[:text_color] || '#ffffff'
       @always_market_price = opts[:always_market_price] || false
+      @needs_token_to_par = opts[:needs_token_to_par] || false
```

Comment:
this is super specific i don't know if i would add it here, probably an override in the game class instead

maybe this is correct, i'm not sure, we need to think about this, but i've never heard of this mechanic before except here

Reply:
Technically I don't think any corp can float without tokens, I could just specialization and make it an always check.
Just this is the only one where you can have corps without tokens afaik.

Reply:
I can't think of any other "needs token to par" games.

Reply:
There are quite a few where you decide how many tokens you want... I'll try and specialize the class then for 1882

Reply:
i thought we were going to delete this?

Reply:
Sorry i hadn't gotten around to it last night

Reply:
As discussed on slack, this is the best available approached